src/paths.ts

```diff
@@ -1,5 +1,6 @@
 import path from 'path';
 import fs from 'fs-extra';
+import { getWorkspaceRoot } from 'workspace-tools';
```

Comment:
if we need only this function, should we use `find-yarn-workspace-root` instead? That's what `workspace-tools` is using under the hood.

Reply:
I think we should use `workspace-tools` wholesale, much of the util code in beachball had been ported over there and has become a sort of dupe.

Reply:
I think I understand what Ken is saying (wholesale?), that we should use `workspace-tools`, and I agree with that. `workspace-tools` also supports rush and pnpm workspaces.